app/scripts/modules/ecs/src/serverGroup/configure/wizard/container/container.component.html

```diff
@@ -1,46 +0,0 @@
-<div class="container-fluid form-horizontal" ng-controller="ecsContainerImageController as containerImageController">
-  <div class="form-group">
-    <div class="col-md-4 sm-label-right">Container Image <help-field key="ecs.dockerimage"></help-field></div>
-    <div class="col-md-8">
-      <ui-select name="imageSelect" ng-model="$ctrl.command.imageDescription" class="form-control input-sm">
-        <ui-select-match placeholder="Please type your search...">{{ $select.selected.imageId }}</ui-select-match>
-        <ui-select-choices
-          repeat="image in $ctrl.command.backingData.filtered.images"
-          group-by="containerImageController.groupByRegistry"
```

Comment:
Does the new react based ui group by registry, and allow searching and pagination? The UI should be usable when you have thousands of tags in your repository. I'm not sure if there are react based examples from other providers for scalable image selection

Reply:
How is this one being addressed?  Is this something we need to address in a different PR too?

Reply:
This logic doesn't exist in the TaskDefinition.tsx as well. It's something I need to look into. If it's not a blocker, I'd suggest merging these changes in, and I'll address these 2 issues in the next PR.